compiler/src/formatting/format.re

```diff
@@ -6,6 +6,22 @@ open Grain_diagnostics;
 
 module Doc = Res_doc;
 
+type iterator_item_type =
+  | Iterated_List_Pattern
+  | Iterated_Record_Pattern
+  | Iterated_Record
+  | Iterated_Type_Constructor
+  | Iterated_Patterns
+  | Iterated_Match_Item
+  | Iterated_Data_Declarations
+  | Iterated_Record_Labels
+  | Iterated_Tuple_Expression
+  | Iterated_Array_Expression
+  | Iterated_Type_Items
+  | Iterated_Tuple_Constructor
+  | Iterated_Enum
+  | Iterated_Record_Data
+  | Iterated_Value_Bindings;
```

Comment:
```suggestion
  | IteratedListPattern
  | IteratedRecordPattern
  | IteratedRecord
  | IteratedTypeConstructor
  | IteratedPatterns
  | IteratedMatchItem
  | IteratedDataDeclarations
  | IteratedRecordLabels
  | IteratedTupleExpression
  | IteratedArrayExpression
  | IteratedTypeItems
  | IteratedTupleConstructor
  | IteratedEnum
  | IteratedRecordData
  | IteratedValueBindings;
```
In this file, we've been using snake-case for the type names (because ocaml needs them lowercase) and variants are PascalCase

Reply:
Sorry, agreed!  I've fixed them in one go locally rather than accept your commit suggestions so I don't trigger a thousand CI jobs.

Reply:
Found how to batch it :)